project/plugins.sbt

```diff
@@ -7,3 +7,5 @@ addSbtPlugin("org.scoverage" % "sbt-scoverage" % "2.0.6")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.0")
 
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.10.0")
+
+addDependencyTreePlugin
```

Comment:
Do we want this plugin in the codebase? I'm not against it, and find it incredibly useful for development, but do we want it in the committed source?

Reply:
idc, up to you